libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1367,6 +1367,25 @@ uint16_t AP_GPS::gps_yaw_cdeg(uint8_t instance) const
     return yaw_cd;
 }
 
+void AP_GPS::send_mavlink_gps_status(mavlink_channel_t chan)
+{
+#if GPS_SATELITES_INFO_ENABLED
+    mavlink_gps_status_t msg{};
```

Comment:
```suggestion
#if AP_GPS_SEND_MAVLINK_GPS_STATUS_ENABLED
void AP_GPS::send_mavlink_gps_status(mavlink_channel_t chan)
{
    mavlink_gps_status_t msg{};
```

 - correct spelling of satellite
 - move compilation guards to outside of function
 - make the define name more descriptive